### PR TITLE
Update WHSSDatlas_v2.jsonld

### PR DESCRIPTION
**Pull request by aeidi89**

### CODE REVIEW

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld [hunk 1]
```diff
@@ -15,8 +15,8 @@
   "copyright": null,
   "custodian": null,
   "description": null,
-  "digitalIdentifier": null,
-  "fullDocumentation": null,
+  "digitalIdentifier": "https://scicrunch.org/resolver/RRID:SCR_017124",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld]: This has to be:

```
"digitalIdentifier": {"@id": "https://scicrunch.org/resolver/RRID:SCR_017124"},
```

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld [hunk 2]
```diff
@@ -266,20 +266,23 @@
 		],
 		"ontologyIdentifier": null
 	},
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Kjonigsen et al., 2015.",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld]: it is better to state the full citation. you don't want people to search for the papers (at least state also the DOI)

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld [hunk 3]
```diff
@@ -266,20 +266,23 @@
...
   "keyword": null,
   "license": {
-    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBySa4.0"
+    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBy4.0"
   },
   "majorVersionIdentifier": null,
   "ontologyIdentifier": null,
   "otherContribution": null,
-  "relatedPublication": null,
+  "relatedPublication": [
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld:282]: This need to be:
```
"relatedPublication": [
	{"@id": "https://doi.org/10.1016/j.neuroimage.2014.04.001"},
	{"@id": "https://doi.org/10.1016/j.neuroimage.2014.10.017"}  
	],
```

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld (hunk 1 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld]: Fixed.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld (hunk 2 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld]: Full citation has been added.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld (hunk 3 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld:282]: Fixed.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld [hunk 4]
```diff
@@ -267,7 +267,9 @@
 		"ontologyIdentifier": null
 	},
   "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Kjonigsen et al., 2015.",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the publications for WHS rat brain atlas v1 (Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2014) Waxholm Space atlas of the Sprague Dawley rat brain. NeuroImage 97, 374-386. DOI: 10.1016/j.neuroimage.2014.04.001) and WHS rat brain atlas v2 (Kjonigsen LJ, Lillehaug S, Bjaalie JG, Witter MP, Leergaard TB (2015) Waxholm Space atlas of the rat brain hippocampal region: Three-dimensional delineations based on magnetic resonance and diffusion tensor imaging. NeuroImage 108, 441-449. DOI: 10.1016/j.neuroimage.2014.12.080
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld]: @aeidi89 thanks. You can format this as markdown. I suggest:

`"Please refer to the atlas by its RRID: SCR_017124, and cite the following publications:\n- Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2014) Waxholm Space atlas of the Sprague Dawley rat brain. NeuroImage 97, 374-386. DOI: https://doi.org/10.1016/j.neuroimage.2014.04.001\n- Kjonigsen LJ, Lillehaug S, Bjaalie JG, Witter MP, Leergaard TB (2015) Waxholm Space atlas of the rat brain hippocampal region: Three-dimensional delineations based on magnetic resonance and diffusion tensor imaging. NeuroImage 108, 441-449. DOI: https://doi.org/10.1016/j.neuroimage.2014.12.080"`

(\n means newline)

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld [hunk 5]
```diff
@@ -280,8 +282,8 @@
   "ontologyIdentifier": null,
   "otherContribution": null,
   "relatedPublication": [
-	"https://doi.org/10.1016/j.neuroimage.2014.04.001",
-	"https://doi.org/10.1016/j.neuroimage.2014.10.017"  
+	{"@id": "https://doi.org/10.1016/j.neuroimage.2014.04.001"},
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld]: please correct the indentation (two more spaces for each list item)

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld (hunk 4 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld]: Thanks @lzehl 
Fixed now.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld (hunk 5 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld]: Done.

**Review by lzehl — APPROVED** [review]

**Pull request merged**